src/components/share/ShareDirective.js

```diff
@@ -21,15 +23,25 @@
               var shortenUrl = scope.options.shortenUrl;
               scope.qrcodegeneratorPath = scope.options.qrcodegeneratorPath;
               scope.mobile = gaBrowserSniffer.mobile;
+              scope.gteIE11 = (gaBrowserSniffer.msie >= 11);
+              scope.gteChrome42 = (gaBrowserSniffer.chrome >= 42);
```

Comment:
Only one scope variable is enough  : 
`iscope.isCopyAllow = (gaBrowserSniffer.msie >= 11) &&(gaBrowserSniffer.chrome >= 42) `